### ISSUE TITLE
Update @ng-select/ng-option-highlight package dependency for angular >10

### ISSUE THREAD

**Issue by antonioseric** [ng-select/ng-select#1824]
**Is your feature request related to a problem? Please describe.**
in npm install i get warings 
```
npm WARN @ng-select/ng-option-highlight@0.0.5 requires a peer of @angular/common@~9.1.0 but none is installed. You must install peer dependencies yourself.
npm WARN @ng-select/ng-option-highlight@0.0.5 requires a peer of @angular/core@~9.1.0 but none is installed. You must install peer dependencies yourself.
npm WARN @ng-select/ng-option-highlight@0.0.5 requires a peer of tslib@^1.10.0 but none is installed. You must install peer dependencies yourself.
```

**Describe the solution you'd like**
No warnings for require old angular version

**Comment by RobinVdBroeck** [comment]
This is resolved in #1795

**Comment by antonioseric** [comment]
It is not released.
